core/src/main/java/org/frankframework/components/LegacyLoader.java

```diff
@@ -99,7 +100,14 @@ private static Manifest convertPomPropertiesToManifest(URL pomProperties) throws
 		int bangSlash = fullUrl.indexOf("!/");
 		if (bangSlash > 0) { // We found a bang-slash, assume it's file
```

Comment:
Ik zit te kijken naar de code en vraag me af of het misschien ligt aan andere assumpties hier.

Dat er iets niet goed is met de aanname dat het een file moet zijn of niet een file is?

Ik wil eigenlijk nog even nadenken hierover en misschien lokaal wat spelen.